src/components/ListDisplay.tsx

```diff
@@ -11,20 +12,36 @@ export declare interface ListDisplayProps {
 }
 
 const ListDisplay = ({activeLink, limit, handleLoadingMore, fetchedData, user}: ListDisplayProps): JSX.Element => (
-  <div>
-    <div className="container space-y-3 max-w-screen-xl mx-auto">
-      {fetchedData.map((item, i) =>
-        <PostList user={user} data={item} key={`${item.full_name}_${i}`}/>)}
-      {fetchedData.length > 0
-        && activeLink !== 'myVotes'
-        && limit <= 100
-        && <button
-          onClick={() => handleLoadingMore()}
-          className="bg-grey hover:bg-lightGrey text-white font-bold py-2 px-4 rounded-xl">
-          Load More
-        </button>
+  <div className="mx-auto max-w-7xl px-[16px] mt-[40px]">
+    <div className='flex flex-col gap-y-[20px]'>
+      <div className='flex items-center gap-x-[10px]'>
+        <img className='w-[26px] h-[26px]' src={popularIcon} alt="Popular" />
+        <h1 className='text-[24px] text-white font-semibold'>Popular repositories</h1>
```

Comment:
```suggestion
        <h1 className='text-[24px] text-white font-semibold'>${activelink} repositories</h1>
```

We should rename the title of the list based on the filter here.